### PR TITLE
search renders meal cards

### PR DESCRIPTION
**Pull request by keithfrederick1**

### CODE REVIEW

File: client/src/MealCard.tsx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  }),
+}));
+//-----------------------------------------------------//
+
+interface CardProps {
+  recipe?: { strMeal: String, idMeal: String, strMealThumb: String}
+}
+
+//the search component should map over the results, creating a meal card for each recipe, 
+export default function MealCard({ recipe }: CardProps) {
```
Comment (copelka) [inline client/src/MealCard.tsx]: use ES6 arrow functions

File: client/src/MealCard.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  const [expanded, setExpanded] = React.useState<boolean>(false);
+  //you only want to make an axios request for the meal that is selected, not for each meal card on the screen, wait until user clicks to assign, and use that recipes name to make the request by passing down the meal prop from state to the Search Youtube component
+  const [meal, setMeal] = React.useState<string>('');//recipe.name
+  const handleExpandClick = () => {
+    console.log(expanded);
```
Comment (copelka) [inline client/src/MealCard.tsx:49]: remove console logs

File: client/src/Search.tsx [hunk 3]
```diff
@@ -22,7 +29,7 @@ const Search = () => {
     axios
       .get<AxiosResponse>(`/routes/search/${ingredients}`)
       .then(({data}: AxiosResponse) => {
-        console.log(data, 19);
+        console.log(data, 32);
```
Comment (copelka) [inline client/src/Search.tsx:32]: remove console logs

**Review by copelka — CHANGES_REQUESTED** [review]

File: client/src/MealCard.tsx (hunk 1 above)
Comment (keithfrederick1) [reply client/src/MealCard.tsx]: i just pushed again

**Pull request merged**